git-playground/mariaines/techinfo

```diff
@@ -1,3 +1,4 @@
 java, python
 visual estudio, notepad++, eclipse
-windows, kali linux
\ No newline at end of file
+windows, kali linux
+langs count: 4
```

Comment:
Maria only has 2 programming langs, please fix

Reply:
Fixed - changes info in line 4 : langs count = 2